src/plugins/discover/public/application/main/hooks/use_adhoc_data_views.ts

```diff
@@ -78,7 +78,7 @@ export const useAdHocDataViews = ({
       return currentDataView;
     }
 
-    const createdDataView = await openConfirmSavePrompt(currentDataView);
+    const createdDataView = currentDataView; // await openConfirmSavePrompt(currentDataView);
```

Comment:
I think we don't need this hook anymore 🥳  ... because it was built to persist an ad-hoc data view for the cases where those don't work. With your PR I think, there is no more 🎉 ... plenty of cleanup possibilities

Reply:
Yeah I think you're right, I'll remove as part of this PR.

Reply:
nice color!
<img width="160" alt="Bildschirmfoto 2023-04-18 um 06 26 58" src="https://user-images.githubusercontent.com/463851/232671577-db8f773c-f397-4950-beae-e285267f61e5.png">